rhine-cassava/src/FRP/Rhine/Clock/Csv.hs

```diff
@@ -0,0 +1,44 @@
+module FRP.Rhine.Clock.Csv (
+  CsvClock,
+  csvClock,
+)
+where
+
+-- bytestring
+import Data.ByteString
+
+-- vector
+import Data.Vector
+
+-- cassava
+import Data.Csv
+
+-- rhine
+
+import Control.Monad.Trans.Except
+import FRP.Rhine (GetClockProxy)
+import FRP.Rhine.Clock
+import FRP.Rhine.Clock.File
+
+{- | Similar to 'File', but ticks at every line of a CSV file,
+and returns a parsed record.
+
+Additionally to an end-of-file error, it can also return a parse error as a 'String'.
+-}
+newtype CsvClock = CsvClock (File String (Vector Record))
```

Comment:
Naming: Better call it `Csv`?

Reply:
This is a bit inconsistent in the code base. I think I've sometimes called something e.g. `SelectClock` to avoid clashing with `Select` (an obscure monad transformer). But in principle it would be nicer not to have `Clock` in the name every time.

Reply:
`Csv` clashes prominently with a type of the same name from `cassava`. I could call it `CsvRow` or something like thit.

Reply:
`CsvRecord`? And a corresponding better name for the parsing variant?